### PR TITLE
BUGZ-1022: Restore avatar theft functionality to Lilypad

### PR DESCRIPTION
**Pull request by SimonWalton-HiFi**
https://highfidelity.atlassian.net/browse/BUGZ-1022

Mostly combines https://github.com/highfidelity/hifi/pull/15930 & https://github.com/highfidelity/hifi/pull/15574.
Regarding Tony's comment https://github.com/highfidelity/hifi/pull/15574#pullrequestreview-239202210: I attempted using Array.prototype.find (or findIndex) but they do not appear to be supported by Qt's JS as they are recent language additions. Instead I've used Array.prototype.every, which is a little cleaner than just a loop.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20444/HighFidelity-Beta-PR15954-63c3452.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20444/HighFidelity-Beta-Interface-PR15954-63c3452.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20444/HighFidelity-Beta-PR15954-63c3452.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20444/HighFidelity-Beta-Interface-PR15954-63c3452.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20444/HighFidelity-Beta-PR15954-63c3452.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20444/HighFidelity-Quest-Beta-PR15954-63c3452.apk)

File: scripts/system/clickToAvatarApp.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 (function () {
     var tablet = Tablet.getTablet("com.highfidelity.interface.tablet.system");
+    
+    
     this.clickDownOnEntity = function (entityID, mouseEvent) {
-        tablet.loadQMLSource("hifi/AvatarApp.qml");
+        var scripts = ScriptDiscoveryService.getRunning();
+       
+        var runningSimplified =
+            !scripts.every(function(item){ return item.name !== "simplifiedUI.js"; });
```
Comment (howard-stearns) [inline scripts/system/clickToAvatarApp.js]: I know this gets silly, but per coding standard:
- Space between `function` and opening paren.
- Space between closing paren and opening brace.
- Arguable (as I prefer your style for short bodies but I imagine that others will disagree), put the body on a separate line, with the previous line ending in the open brace, and the following line ending ain a close brace, close paren, semicolon.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20450/HighFidelity-Beta-PR15954-8fb28bf.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20450/HighFidelity-Beta-Interface-PR15954-8fb28bf.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20450/HighFidelity-Beta-PR15954-8fb28bf.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20450/HighFidelity-Quest-Beta-PR15954-8fb28bf.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20450/HighFidelity-Beta-PR15954-8fb28bf.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20450/HighFidelity-Beta-Interface-PR15954-8fb28bf.exe

**Pull request merged**